### ISSUE TITLE
Android 13 Notification Player Images are pixelated and blurry

### ISSUE THREAD

**Issue by sleyland** [doublesymmetry/react-native-track-player#1934]
**Describe the Bug**
When viewing the notification player (in either the notification pull down or the lock screen) the image which is set into the player appears blurry and pixelated.

**Steps To Reproduce**
* Create an Android 13 (v33) compatible AVD (a Pixel_5_API_33 AVD was used to produce the issue)
* Start up the example app from the `react-native-track-player` project
* Slide down the notification screen and verify that the image appears blurry/pixelated
This issue can be reproduced both on a physical device (with the example app) or on the emulator (with the example app).

**Code To Reproduce**
No additional code is required and the issue can be reproduced from the example app included within the project.

**Replicable on Example App?**
Yes

**Environment Info:**
System:
    OS: macOS 13.2.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 4.25 GB / 32.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.0.0 - ~/.nodenv/versions/16.0.0/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 7.10.0 - ~/.nodenv/versions/16.0.0/bin/npm
    Watchman: 2023.02.13.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8815526
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.18 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.68.5 => 0.68.5
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

**How I can Help**
I have found a similar issue from another repository (https://github.com/androidx/media/issues/168) which talks about the way in which the bitmap is set into the player and how the usage of a *large icon* can potentially cause the pixelation issue to occur. 

It is possible that this issue actually exists within the https://github.com/doublesymmetry/KotlinAudio code instead and could be related to *need* to upgrade/support Media3?

I'm unfamiliar with the Android code, but prepared to help out with a bit of direction.

**Comment by afkcodes** [comment]
I see that 1.0 is released

**Comment by jspizziri** [comment]
@craigmclean39 happy to review a PR.

**Comment by puckey** [comment]
This will be fixed by the following KotlinAudio pr: https://github.com/doublesymmetry/KotlinAudio/pull/85

**Comment by gavrichards** [comment]
Has this been merged into an RC of v4? Sorry, I'm struggling to tell. I can see the KotlinAudio PR is merged, just not sure if that's made its way into RNTP.

**Comment by gavrichards** [comment]
Oh great! Should this issue be closed?

**Comment by gavrichards** [comment]
> > Oh great! Should this issue be closed?
> 
> I sometimes wait for confirmation that it was actually resolved :) Want to confirm?

Yes. I haven't seen it on my device, but we've had reports from customers, so I'll push a new version to production and get them to confirm. Will report back.
